lib/modules/datasource/glasskube-packages/index.ts

```diff
@@ -0,0 +1,77 @@
+import urljoin from 'url-join';
+import { logger } from '../../../logger';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import {
+  GlasskubePackageManifest,
+  GlasskubePackageManifestYaml,
+  GlasskubePackageVersions,
+  GlasskubePackageVersionsYaml,
+} from './schema';
+
+export class GlasskubePackagesDatasource extends Datasource {
+  static readonly id = 'glasskube-packages';
+  static readonly defaultRegistryUrl =
+    'https://packages.dl.glasskube.dev/packages';
+
+  override defaultRegistryUrls = [
+    GlasskubePackagesDatasource.defaultRegistryUrl,
+  ];
+
+  constructor() {
+    super(GlasskubePackagesDatasource.id);
+  }
+
+  override async getReleases({
+    packageName,
+    registryUrl,
+  }: GetReleasesConfig): Promise<ReleaseResult | null> {
+    let versions: GlasskubePackageVersions;
+    const result: ReleaseResult = { releases: [] };
+
+    try {
+      const response = await this.http.get(
+        urljoin(registryUrl!, packageName, 'versions.yaml'),
+      );
+      versions = GlasskubePackageVersionsYaml.parse(response.body);
+    } catch (error) {
+      logger.error(
+        { error, registryUrl },
+        'Failed to request releases from Glasskube packages datasource',
+      );
+      return null;
+    }
+
+    result.releases = versions.versions.map((it) => ({
+      version: it.version,
+      isStable: it.version === versions.latestVersion,
```

Comment:
Are all versions except the latest one considered pre releases? 

Think `-SNAPSHOT` for Maven/Gradle for example

Reply:
No, the `latestVersion` typically represents the newest version that has undergone an additional manual vetting step after being added to the repository. We generally do not include prerelease versions.

Should I change this so that `isStable` is always false?

Reply:
You can simply remove it as the system will assume that the version is stable